docs/review_criteria.md

```diff
@@ -56,11 +56,11 @@ The authors should clearly state what problems the software is designed to solve
 
 #### Installation instructions
 
-There should be a clearly-stated list of dependencies. Ideally these should be handled with an automated package management solution.
+Software dependencies should be clearly documented and their installation handled by an automated proceedure. Where possible, software installation should be managed with a package manager. However, this criterion depends upon the maturity and availability of software packaging and distribution in the programming language being used. For example, Python packages should be `pip install`able, and have adopted [packaging conventions](https://packaging.python.org), while Fortran submissions with a Makefile may be sufficient.
```

Comment:
for python: does this mean we enforce pypi packaging and not allow conda-forge?

Reply:
I would lean toward listed both as examples ("Python packages should be `pip install`able and/or `conda install`able with the conda-forge channel...") to be clear but certainly being installable with conda-forge should be ok. 

Is it ok for a package to say "clone this from github and install locally with `pip install -e .`?

Reply:
> Is it ok for a package to say "clone this from github and install locally with pip install -e .?

@kthyng I can't think of a reason why a dev install would be a good default choice compared to hosting the package on pypi or conda-forge.

Reply:
@faroit I agree, but I think we may see this come through. I guess it is covered by what it says above, though (about needing to be pip installable).

Reply:
No I take it back, @faroit, @arfon I think we should specify that it can't be dev installable only since people might think it is pip installable since it uses "pip". This is another thing a possibly not-software-oriented person would have to learn to do though.

Reply:
There's also a branding/namespace issue, in that if we publish a paper on a Python package and `pip install foo` works, the reader should be getting the package they're reading about. If we only check that local installs work, a different package (now or in the future) could take that name in PyPI. Of course we can't enforce this across all packaging systems, but one or two are usually pervasive within any given community.

This gets more complicated fast for software that is not single-language. I'd also note that sometimes software is packaged for binary distribution, but developer installs are really hard. That's enough for typical users, but unhealthy for projects that need users to wet their feet by contributing. Should our requirements say that software should be packaged (at least one relevant place) *and* that developer installs are easy to follow/play well with others? (For example, they shouldn't require a custom Docker image to build.)